celery_utils/logged_task.py

```diff
@@ -36,6 +36,14 @@ def on_retry(self, exc, task_id, args, kwargs, einfo):
         """
         Capture the exception that caused the task to be retried, if any.
         """
-        super(LoggedTask, self).on_retry(exc, task_id, args, kwargs, einfo)
         if exc is not None:
             log.warning('[{}] retried due to {}'.format(task_id, einfo.traceback))
+        super(LoggedTask, self).on_retry(exc, task_id, args, kwargs, einfo)
+
+    def on_failure(self, exc, task_id, args, kwargs, einfo):
```

Comment:
@mulby Interesting.  So we currently haven't enabled logging from celery itself?  When @jcdyer and I looked into this earlier, we looked at what was outputted in Splunk and weren't satisfied with the amount of information we found.  I like how their standard logging is prefixed, consistent, and includes the task's name (so they're easy to find).

Should we experiment with enabling celery's >= Error logs in production (behind a waffle switch?) and make sure it doesn't overwhelm our Splunk quota?

For logging retries, however, their log setting is at the "Info" level.  So it makes sense to keep this logging base class in tact for Retry logging, right?